cmake/projects/tommath/hunter.cmake

```diff
@@ -0,0 +1,27 @@
+# cmake/projects/Example/hunter.cmake
+
+# !!! DO NOT PLACE HEADER GUARDS HERE !!!
+
+# Load used modules
+include(hunter_add_version)
+include(hunter_download)
+include(hunter_pick_scheme)
+include(hunter_cacheable)
+
+# List of versions here...
+hunter_add_version(
+    PACKAGE_NAME
+    tommath
+    VERSION
+    "1.0-p1"
```

Comment:
Same as [here](https://github.com/ruslo/hunter/pull/639/files#r103096124), can you please make it `v1.0-hunter`?